modules/appeals_api/spec/support/schema_helpers.rb

```diff
@@ -15,4 +17,58 @@ def read_schema(filename, schema_version = 'v1')
       )
     )
   end
+  # rubocop:disable Layout/LineLength
+
+  def find_max_lengths(filename, _string_to_find, schema_version = 'V2')
```

Comment:
This is awesome. I do wonder if this might become difficult to maintain as version changes occur, but probably not more work than writing new specs etc for new versions.